optaplanner-docs/src/main/asciidoc/DesignPatterns/DesignPatterns-chapter.adoc

```diff
@@ -11,93 +11,73 @@
 [[designPatternsIntroduction]]
 == Design Patterns Introduction
 
-These design patterns list and solve common design challenges.
-
+OptaPlanner design patterns are generic reusable solutions to common challenges in the model or architecture of projects that perform constraint solving. The design patterns in this section list and solve common design challenges.
 
 [[domainModelingGuide]]
-== Domain Modeling Guide
-
-A good model can greatly improve the success of your planning implementation.
-These guidelines help to design a good model:
-
-. *Draw a class diagram* of your domain model. _Normalize_ it get rid of duplicate data.
-** Write down some sample instances for each class.
-*** For example in employee rostering, the samples for the `Employee` class are _Ann_, _Bert_ and _Carl_.
-
-. *Determine which relationships (or fields) change during planning*. Color them orange.
-Often there is only one such relationship (or field).
-One side of these relationships will become a planning variable later on.
-** For example in employee rostering, the `Shift` to `Employee` relationship changes during planning,
-so it is orange.
-Other relationships, such as from `Employee` to `Skill`, are immutable during planning
-because Planner can't decide to give an employee an extra skill.
-
-. If there are multiple relationships (or fields): *check for <<shadowVariable,shadow variables>>*.
-A shadow variable does change during planning,
-but its value can be calculated based on one or more genuine planning variables, without dispute.
-Color those shadow relationships (or fields) purple.
-** Only one side of a bi-directional relationship can be a genuine planning variable,
-the other side will become an <<bidirectionalVariable,inverse relation shadow variable>> later on.
-Keep those relationships in orange.
+== Domain Modeling Guidelines
+
+Follow the guidelines listed in this section to create a well thought-out model that can contribute significantly to the success of your planning.
+
+
+. *Draw a class diagram of your domain model.*
+.. Make sure there are no duplications in your data model and that relationships between objects are clearly defined.
+
+.. Create sample instances for each class. For example, in the Employee Rostering starter application, the samples for the `Employee` class are `Ann`, `Bert`, and `Carl`.
+
+. *Determine which relationships (or fields) change during planning and color them orange.* One side of these relationships will become a planning variable later on. For example, in the Employee Rostering starter application, the `Shift` to `Employee` relationship changes during planning, so it is orange. However, other relationships, such as from `Employee` to `Skill`, are immutable during planning
```

Comment:
will become -> becomes? (original text also wrote "will become", but we should use present tense?)

Reply:
Many sections write "For example in Employee Rostering ..." or "For example in Vehicle Routing ...". Although there are code examples in optaplanner-examples for those use cases, many of the sentences using  "For example in Employee Rostering ..." do not refer to anything that is actually implemented in those optaplanner-examples. It just could it - it could make sense depend on the reader's goals.
I'd avoid adding the "starter application" suffix, as it implies that the example (= ambiguity which one) actually implements it.

Reply:
will become - it is appropriate to use future tense when we are truly talking about something that 'will' happen in the future. But I see your point and in this case we can use present tense so I 'will' change :-)

I will also remove 'starter application throughout.

Again, the 'for examples' are part of the original text. Do you want to remove  them completely. or alter them in some other way?

Reply:
Thanks for removing "starter application" throughout.
I'd keep "For example", as it's consistent style use across the user guide to go from explaining a topic to illustrating it with an example